web/sites/default/config/views.view.frontpage.yml

```diff
@@ -186,7 +186,7 @@ display:
             next: 'Next ›'
             first: '« First'
             last: 'Last »'
-          quantity: 9
+          quantity: 4
```

Comment:
This view could be disabled if it's not. We use a page manager page for the frontpage.